src/stryker-net/mutators.pug

```diff
@@ -0,0 +1,114 @@
+extends ../layout/section-layout.pug
+
+block content
+    :markdown-it
+        Stryker supports a variety of mutators, which are listed below. Do you have a suggestion for a (new) mutator? Feel free to create an [issue](https://github.com/stryker-mutator/stryker-net/issues)!
```

Comment:
Will you still have the supported mutators in the handbook? In my experience it's been a bit of a PITA to keep the same documentation in two places